content/fast/using-lighthouse-ci-to-set-a-performance-budget/index.md

```diff
@@ -127,46 +131,57 @@ script:
  - webpack</td>
 </pre>
 
+The YAML file tells Travis to install all the dependencies and build your app.
+Now it’s your turn to **push the example app to your own GitHub repository**.
+If you haven't already, run the following command:
 
-The YAML file tells Travis to install all the dependencies and build your app. Now it’s your turn to **push the example app to your own GitHub repository**.  If you haven't already, run the following command:
-
-<pre class="devsite-terminal devsite-click-to-copy">git push origin master</pre>
+<pre class="devsite-terminal devsite-click-to-copy">
+git push origin master
+</pre>
 
-Click on your repo under Settings in Travis to see your project's Travis dashboard. If everything is cool, you’ll see your build go from yellow to green in a couple of minutes. 🎉
+Click on your repo under Settings in Travis to see your project's Travis
+dashboard. If everything is cool, you’ll see your build go from yellow to green
+in a couple of minutes. 🎉
 
-## 4. Automate Firebase deployment with Travis 
+## 4. Automate Firebase deployment with Travis
 
-In Step 2, we logged into our Firebase account and deployed our app from the command line with `firebase deploy`. In order for Travis to deploy our app to Firebase, we have to authorize it. How do we do that? With a Firebase token. 🗝️🔥
+In Step 2, you logged into our Firebase account and deployed the app from the
+command line with `firebase deploy`. In order for Travis to deploy our app to
+Firebase, you have to authorize it. How do you do that? With a Firebase token.
+🗝️🔥
 
 ### Authorize Firebase
 
-To generate the token run this command: 
+To generate the token run this command:
 
 <pre class="devsite-terminal devsite-click-to-copy">firebase login:ci</pre>
 
-It will open a new tab in a browser window so that Firebase can verify you. After that, look back at the console, and you’ll see your freshly minted token. Copy it and go back to Travis.
+It will open a new tab in a browser window so that Firebase can verify you.
+After that, look back at the console, and you’ll see your freshly minted token.
+Copy it and go back to Travis.
 
-In your project's Travis dashboard, go to More options > Settings > Environment variables. 
+In your project's Travis dashboard, go to More options > Settings > Environment variables.
```

Comment:
```suggestion
In your project's Travis dashboard, go to **More options** > **Settings** > **Environment variables**.
```

Reply:
I think this may look better. Might be worth testing locally to see if you think it looks ok.